net/tstun/mtu.go

```diff
@@ -79,14 +79,34 @@ const (
 	safeTUNMTU TUNMTU = 1280
 )
 
-// MaxProbedWireMTU is the largest MTU we will test for path MTU
-// discovery.
-var MaxProbedWireMTU WireMTU = 9000
+// WireMTUsToProbe is a list of the on-the-wire MTUs we want to
+// probe.
+var WireMTUsToProbe = []WireMTU{
+	WireMTU(WireToTUNMTU(WireMTU(safeTUNMTU))), // Tailscale over Tailscale :)
```

Comment:
If `safeTUNMTU` is already a TUNMTU, should we be explicitly type-casting it to `WireMTU` just to convert it back via `WireToTUNMTU`? Is safeTUNMTU actually a TUNMTU if it's not already accounting for wgHeaderLen?

Reply:
You're right, this is overly complicated. I just have to cast safeTUNMTU to a WireMTU and I'm done.